tests/test_orbit.py

```diff
@@ -10103,3 +10103,488 @@ def test_3d_tol_integration():
+    r1_at_10 = o.r(10.0)
+
+    # Second integration continuing from first
+    t2 = numpy.linspace(10.0, 20.0, 101)
+    o.integrate(t2, MWPotential2014)
+
+    # Check that time array was merged
+    assert len(o.t) == 201, "Time array should have 201 points after continuation"
+    assert numpy.isclose(o.t[0], 0.0), "First time should be 0"
+    assert numpy.isclose(o.t[-1], 20.0), "Last time should be 20"
+    assert numpy.isclose(o.t[100], 10.0), "Middle time should be 10"
+
+    # Check that orbit was merged
+    assert o.orbit.shape[1] == 201, "Orbit should have 201 time points"
+
+    # Check that r at junction is continuous
+    assert numpy.isclose(o.r(10.0), r1_at_10), "r should be continuous at junction"
+
+    # Check that methods work across the full range
+    r_all = o.r(o.t)
+    assert r_all.shape == (201,), "r should work for all times"
+
+    return None
+
+
+def test_orbit_continuation_backward():
+    # Test backward continuation of orbit integration
+    from galpy.orbit import Orbit
+    from galpy.potential import MWPotential2014
+
+    o = Orbit([1.0, 0.1, 1.1, 0.0, 0.1])
+
+    # First integration (forward)
+    t1 = numpy.linspace(0.0, 10.0, 101)
+    o.integrate(t1, MWPotential2014)
+    r1_at_0 = o.r(0.0)
+
+    # Second integration going backward
+    t2 = numpy.linspace(0.0, -10.0, 101)
+    o.integrate(t2, MWPotential2014)
+
+    # Check that time array was merged correctly
+    assert len(o.t) == 201, "Time array should have 201 points after continuation"
+    assert numpy.isclose(o.t[0], -10.0), "First time should be -10"
+    assert numpy.isclose(o.t[-1], 10.0), "Last time should be 10"
+    assert numpy.isclose(o.t[100], 0.0), "Middle time should be 0"
+
+    # Check that orbit was merged
+    assert o.orbit.shape[1] == 201, "Orbit should have 201 time points"
+
+    # Check that r at junction is continuous
+    assert numpy.isclose(o.r(0.0), r1_at_0), "r should be continuous at junction"
+
+    # Check that methods work across the full range
+    r_all = o.r(o.t)
+    assert r_all.shape == (201,), "r should work for all times"
+
+    return None
+
+
+def test_orbit_continuation_different_potential():
+    # Test that continuation doesn't happen with different potential
+    from galpy.orbit import Orbit
+    from galpy.potential import LogarithmicHaloPotential, MWPotential2014
+
+    o = Orbit([1.0, 0.1, 1.1, 0.0, 0.1])
+
+    # First integration
+    t1 = numpy.linspace(0.0, 10.0, 101)
+    o.integrate(t1, MWPotential2014)
+
+    # Second integration with different potential
+    t2 = numpy.linspace(10.0, 20.0, 101)
+    o.integrate(t2, LogarithmicHaloPotential())
+
+    # Check that integration was NOT continued (should have only new times)
+    assert len(o.t) == 101, "Time array should have 101 points (not continued)"
+    assert numpy.isclose(o.t[0], 10.0), "First time should be 10"
+    assert numpy.isclose(o.t[-1], 20.0), "Last time should be 20"
+
+    return None
```

Comment:
Now that using a different potential is allowed, this test is no longer relevant